eZ/Publish/API/Repository/Tests/Regression/EZP20018VisibilityTest.php

```diff
@@ -33,7 +33,7 @@ public function testSearchForHiddenContent()
         $this->assertEquals(0, $results1->totalCount);
         $this->assertCount(0, $results1->searchHits);
 
-         // Hide "Images" Folder
+        // Hide "Images" Folder
```

Comment:
Don't know why this and similar indentation issues were matched by `braces` rule, but it's worth to align code properly.